updatezinfo.py

```diff
@@ -17,9 +19,19 @@ def main():
 
     if not os.path.isfile(metadata['tzdata_file']):
         print("Downloading tz file from iana")
-        request.urlretrieve(os.path.join(metadata['releases_url'],
-                                         metadata['tzdata_file']),
-                            metadata['tzdata_file'])
+        for x in range(1, 6):
```

Comment:
Ok, I can remove this commit when I get back. I just thought it would be nice not to have to restart the CI. I also did not like seeing the red x in this pr :-p